src/components/ErrorCard/ErrorCard.js

```diff
@@ -0,0 +1,29 @@
+import { View } from 'react-native';
+import React from 'react';
+import { NativeUiText } from '..';
```

Comment:
Please use direct imports instead of index file.

like
```suggestion
import { NativeUiText } from '../NativeUiText';
```